tests/testthat/test-colors.R

```diff
@@ -119,3 +119,19 @@ test_that("decode_color_keyword", {
   expect_error(decode_color_keyword(" orange "))
   expect_error(decode_color_keyword(NA))
 })
+
+test_that("parseCssColors() handles incoming NA values sensibly", {
+  expect_error(parseCssColors(NA))
+  expect_identical(
+    parseCssColors(NA_character_, mustWork = FALSE),
```

Comment:
I think we should also keep the old version of the test, with `NA_logical_`, because it seems likely that users will pass in regular old `NA`. The logic for `parseCssColors` will also have to be updated to handle this.